skycultures/arabian_peninsula/constellationship.fab

```diff
@@ -0,0 +1,35 @@
+﻿100 1 72607 75097 
+200 6 54061 53910 53910 58001 58001 59774 59774 62956 62956 65378 65378 67301 
+300 4 746 3179 3179 4427 4427 6686 6686 8886 
+400 4 102098 100453 100453 95947 102488 100453 100453 97165 
+2502 1 113881 113963 
+2504 1 677 1067 
+2550 10 3881 3231 3231 2912 2912 3031 3031 3693 3693 5454 5454 5571 5571 6193 6193 6315 6315 5447 5447 3881 
+2602 1 9884 8903 
+2603 3 12719 13209 13209 13061 13061 12719 
+2604 2 17608 17608 18163 18163 
+2801 1 37279 36188 
+4501 1 33579 34444 
+2803 3 42806 42556 42556 42911 43004 43004 
+2804 1 47908 47508 
+2805 3 50335 50583 50583 49583 49583 49669 
+2806 1 54872 54879 
+2808 4 63608 63090 63090 61941 61941 60129 60129 57757 
+2810 3 69701 69427 69427 69974 70784 70784 
+2901 1 72622 74785 
+2902 2 78820 78401 78401 78265 
+2906 1 85927 85696 
+2999 15 74785 78820 78820 80112 72622 78265 78265 80112 78401 80112 80112 80763 80763 81266 81266 82396 82396 82514 82514 82729 82729 84143 84143 86228 86228 87073 87073 86670 86670 85927 
+3005 5 97290 96760 96760 95503 95503 95564 95564 96440 96440 97290 
+3099 14 89931 90185 90185 93506 93506 92041 92041 89931 88635 89642 89642 90185 90185 89931 89931 88635 92855 93864 93864 93506 93506 92041 92041 92855 89931 90496 90496 92041 
+3199 2 100064 100310 100310 100345 
+3299 2 102618 103045 103045 104459 
+3399 2 106278 106786 106786 107382 
+3499 2 110395 110960 110960 110672 110960 111497 110395 110672 110395 111497 
+4201 4 26207 26176 26176 26366 26366 26207 26444 26444 
+4206 1 31681 32362
+4213 4 25336 25336 27989 27989 24436 24436 27366 27366
+4202 2 25930 26311 26311 26727 
+2650 2 9884 8796 8796 10064 
+4299 10 26207 26487 25336 26487 26487 27989 25336 25930 25930 26311 26311 26727 26727 27366 27366 24436 24436 25930 27989 26727 
+ 
```

Comment:
Github tells me "no newline at end of file". Please add one.

Reply:
Fixed, last line had a blank charactes.